src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/commands.py

```diff
@@ -78,17 +82,28 @@
         # with s.group('sql db restore-point') as c:
         #     c.command('list', 'list_restore_points')
 
-        # TDE will not be included in the first batch of GA commands
-        # with s.group('sql db transparent-data-encryption') as c:
-        #     c.command('create', 'create_or_update_transparent_data_encryption_configuration')
-        #     c.command('show-configuration', 'get_transparent_data_encryption_configuration')
-        #     c.command('show-activity', 'list_transparent_data_encryption_activity')
-
         # Service tier advisor will not be included in the first batch of GA commands
         # with s.group('sql db service-tier-advisor') as c:
         #     c.command('list', 'list_service_tier_advisors')
         #     c.command('show', 'get_service_tier_advisor')
 
+    transparent_data_encryptions_operations = create_service_adapter(
+        'azure.mgmt.sql.operations.transparent_data_encryptions_operations',
+        'TransparentDataEncryptionsOperations')
+    with ServiceGroup(__name__, get_sql_database_transparent_data_encryptions_operations,
+                      transparent_data_encryptions_operations, custom_path) as s:
+        with s.group('sql db tde') as c:
+            c.command('update', 'create_or_update')
+            c.command('show', 'get')
+
+    transparent_data_encryption_activities_operations = create_service_adapter(
+        'azure.mgmt.sql.operations.transparent_data_encryption_activities_operations',
+        'TransparentDataEncryptionActivitiesOperations')
+    with ServiceGroup(__name__, get_sql_database_transparent_data_encryption_activities_operations,
+                      transparent_data_encryption_activities_operations, custom_path) as s:
+        with s.group('sql db tde-activity') as c:
+            c.command('list', 'list_by_configuration')
```

Comment:
With there be additional tde-activity commands?  If not, this should be up-leveled `sql db list-tde-activity`.

Reply:
There will not be additional tde-activity commands. I'll change it to your suggestion.

Reply:
Or `sql db tde list-activity` ?

Reply:
Since there is `sql db tde` group already

Reply:
@jaredmoo yes, that makes even more sense!